solutions/gold/rmi-17-Hangman2.mdx

```diff
@@ -0,0 +1,147 @@
+---
+id: rmi-17-Hangman2
+source: RMI 2017 Day 1
+title: Hangman 2
+author: Justin Ji
+---
+
+<Spoiler title = "Hint">
+
+Consider breaking the problem into two cases: $n < k$ and $n \geq k$.
```

Comment:
add a sequel for this

Reply:
by sequel, do you mean another hint that sorta points them to how they should handle each case?

Reply:
yeah
like i read this
"ok??? n<k and????"

Reply:
yea ^

Reply:
ok I added a hint. Hopefully it isn't as much of a cliffhanger now lol